README.md

```diff
@@ -15,6 +15,7 @@ cd bahmni-docker
 export OPENMRS_MODULES_PATH="/path/to/the/openmrs/modules"
 export BAHMNI_CONFIG_PATH="/path/to/the/bahmni/config"
 export OPENMRS_CONFIG_PATH="/path/to/the/openmrs/config"
+export BAHMNI_HOME="/path/to/the/bahmni/home"
 ```
 
```

Comment:
Could you add a section to set the timezone as well?
I believe that is something that we should implement across all containers.

Reply:
Sure will do.